Task "serve-global" in Makefile generated by pelican-quickstart requires port, but it is not defined

The task "serve-global" requires that the PORT variable is defined, but it is defined nowhere in the Makefile. 

The problem is in the Makefile template (pelican/tools/templates/Makefile.jinja2):

```
serve-global:
ifdef SERVER
  $(PELICAN) -l $(INPUTDIR) -o $(OUTPUTDIR) -s $(CONFFILE) $(PELICANOPTS) -p $(PORT) -b $(SERVER)
else
  $(PELICAN) -l $(INPUTDIR) -o $(OUTPUTDIR) -s $(CONFFILE) $(PELICANOPTS) -p $(PORT) -b 0.0.0.0
endif
```

The "serve" task checks for port, but "serve-global" only checks for the server. Maybe the solution is a nested check for port (as done in "serve").

Error message is as follows:

```
pelican -l /home/nik/pelican/content -o /home/nik/pelican/output -s /home/nik/pelican/pelicanconf.py  -p  -b 0.0.0.0
usage: pelican [-h] [-t THEME] [-o OUTPUT] [-s SETTINGS] [-d] [-v] [-q] [-D]
               [--version] [-r]
               [--print-settings [SETTING_NAME [SETTING_NAME ...]]]
               [--relative-urls] [--cache-path CACHE_PATH] [--ignore-cache]
               [-w SELECTED_PATHS] [--fatal errors|warnings]
               [--logs-dedup-min-level {DEBUG,INFO,WARNING,ERROR}] [-l]
               [-p PORT] [-b BIND]
               [path]
pelican: error: argument -p/--port: expected one argument
make: *** [Makefile:65: serve-global] Fehler 2
```

--- a/pelican/tools/templates/Makefile.jinja2
+++ b/pelican/tools/templates/Makefile.jinja2
@@ -50,6 +50,14 @@ ifeq ($(RELATIVE), 1)
 	PELICANOPTS += --relative-urls
 endif
 
+SERVER ?= "0.0.0.0"
+
+PORT ?= 0
+ifneq ($(PORT), 0)
+	PELICANOPTS += -p $(PORT)
+endif
+
+
 help:
 	@echo 'Makefile for a pelican Web site                                           '
 	@echo '                                                                          '
@@ -100,12 +108,7 @@ else
 endif
 
 serve-global:
-ifdef SERVER
-	$(PELICAN) -l $(INPUTDIR) -o $(OUTPUTDIR) -s $(CONFFILE) $(PELICANOPTS) -p $(PORT) -b $(SERVER)
-else
-	$(PELICAN) -l $(INPUTDIR) -o $(OUTPUTDIR) -s $(CONFFILE) $(PELICANOPTS) -p $(PORT) -b 0.0.0.0
-endif
-
+	$(PELICAN) -l $(INPUTDIR) -o $(OUTPUTDIR) -s $(CONFFILE) $(PELICANOPTS) -b $(SERVER)
 
 devserver:
 ifdef PORT

